package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "@uniswap/smart-order-router",
+  // TODO: bump patch version
```

Comment:
you probably still need that here right? happy to review when reayd

Reply:
will let you know

Reply:
done